toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -811,7 +811,7 @@ func updateInitramfsForEncrypt(installChroot *safechroot.Chroot) (err error) {
 	return
 }
 
-func updateFstab(installRoot string, config configuration.SystemConfig, installMap, mountPointToFsTypeMap, mountPointToMountArgsMap map[string]string, hidepidEnabled bool) (err error) {
+func updateFstab(installRoot string, config configuration.SystemConfig, installMap, mountPointToFsTypeMap, mountPointToMountArgsMap, partIDToDevPathMap map[string]string, hidepidEnabled bool) (err error) {
```

Comment:
No, swap partition first of all does not have a mount point. And what needs to be added to the fstab file is the device path instead of the mount point.